src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysSize = 3;
+        int resultIndex = 2;
+        String[] type = new String[arraysSize];
+        int[] amount = new int[arraysSize];
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
Use try-with-resources to ensure that the `BufferedReader` is closed after use to avoid resource leaks.